src/client/datascience/prereleaseChecker.ts

```diff
@@ -28,7 +28,7 @@ export class PreReleaseChecker implements IExtensionSingleActivationService {
                 const dontAsk = this.globalState.get(PRERELEASE_DONT_ASK_FLAG, false);
 
                 // Ask user if the version is not prerelease
-                if (!isPreRelease && dontAsk && this.appEnv.channel === 'insiders') {
+                if (!isPreRelease && !dontAsk && this.appEnv.channel === 'insiders') {
```

Comment:
Default is false. So it would never ask. Doh.

Reply:
The old "negative embedded in the variable name" trick ;)